Src/Campus.Master.API/UI/src/app/modules/auth/registration/registration.component.ts

```diff
@@ -82,33 +67,18 @@ export class RegistrationComponent implements OnInit, OnDestroy {
   }
 
   submit() {
-    this.spinner = true;
-    this.submitted = true;
-    if (this.registerForm.invalid) {
-      return;
-    }
     const registerUser: RegisterUser = {
       fullName: this.registerForm.value.first_name + ' ' + this.registerForm.value.last_name,
```

Comment:
why fullName, you can path firstName and lastName, in that case you can just take a value from a form

Reply:
Will be refactored in further pr